snap/snapcraft.yaml

```
@@ -145,6 +145,8 @@ parts:
       craftctl set version="${version}"
       [ -n "$(echo $version | grep "post")" ] && grade=devel || grade=stable
       craftctl set grade="${grade}"
+      # Set up the rust compiler
+      rustup default 1.79
```

Let's go with the latest :smile: 
```suggestion
      rustup default 1.80
```

---

I'm intentionally going with the minimum version to support the libraries - otherwise I'd just put in `stable` :-)